java/runtime/src/main/java/org/ray/runtime/util/generator/RayCallGenerator.java

```diff
@@ -213,13 +231,15 @@ private void buildCalls(int numParameters, boolean forActor,
   }
 
   /**
-   * Build the `Ray.callPy` or `Ray.createPyActor` methods.
+   * Build the `Ray.callPyActor` or `Ray.createPyActor` methods.
```

Comment:
Shouldn't we align the comment with `buildCalls`?

Reply:
I fixed the comments for buildPyCalls.